File: Assets/PACKT_Scripts/zombieWalkTest01.js
#pragma strict
var zombieControl : Animator;
var walkBool : boolean = false;

function Start () 
{
	zombieControl = GetComponent(Animator);
}

function Update () 
{
	if(Input.GetKeyDown("b"))
	{
		walkBool = !walkBool;
		zombieControl.SetBool("SwitchWalk", walkBool);
	}

}

File: Assets/PACKT_Scripts/zombieWalkTest01.cs
using UnityEngine;
using System.Collections;

public class zombieWalkTest01 : MonoBehaviour 
{
	public Animator zombieControl;
	public bool walkBool;

	void Start () 
	{
		zombieControl = GetComponent<Animator>();
	}
	
	void Update () 
	{
		if(Input.GetKeyDown("b"))
		{
			walkBool = !walkBool;
			zombieControl.SetBool("SwitchWalk", walkBool);
		}
	}
}
